ui/app/serializers/clients/activity.js

```diff
@@ -29,12 +32,50 @@ export default class ActivitySerializer extends ApplicationSerializer {
     });
   }
 
-  // For 1.10 release naming changed from 'distinct_entities' to 'entity_clients' and
+  // for vault usage - vertical bar chart
+  flattenByMonths(payload, isNewClients = false) {
+    if (isNewClients) {
+      return payload.map((m) => {
+        return {
+          month: m.timestamp,
+          entity_clients: m['new_clients']['counts']['entity_clients'],
```

Comment:
Since the selectors are static strings, you can just access like
```suggestion
          entity_clients: m.new_clients.counts.entity_clients,
```
and ignoring the linting. Alternatively, we can make this a little more concise by removing the outer `isNewClients` check and doing:
```js
const dataKey = isNewClients ? 'new_clients.counts' : 'counts';
return payload.map(m => {
  const flat = {
    entity_clients: m[dataKey].entity_clients,
    ...
  }
  if (isNewClients) {
    flat.new_clients = { ... }
  }
}
```